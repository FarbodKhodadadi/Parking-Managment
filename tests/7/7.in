request_spot Ford_F150
assign_spot 53
assign_spot 25
request_spot Mazda_3
pass_time 5
pass_time 1
pass_time 4
checkout 25
request_spot Mazda_3
request_spot Ford_F150
checkout 53
request_spot Mazda_3
