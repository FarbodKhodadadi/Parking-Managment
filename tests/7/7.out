25: CCTV 530 150
Spot 53 is occupied now.
Spot 25 is occupied now.
30: covered 830 95
Spot 25 is free now.
Total cost: 2030
30: covered 830 95
25: CCTV 530 150
Spot 53 is free now.
Total cost: 1780
30: covered 830 95
53: covered 830 95
